plugin/pxr/maya/lib/usdMaya/testenv/testUsdExportPackage.py

```diff
@@ -24,7 +23,6 @@
 
 from pxr import Usd, UsdGeom
 
-
```

Comment:
@mattyjams  This test is still failing in Python3 on Windows:

The reason it fails is because GetFileNames which is simply a list of strings is different from the expected list.

```
zipFile = Usd.ZipFile.Open(usdFile)
fileNames = zipFile.GetFileNames()
self.assertEqual(fileNames, [
    "MyAwesomePackage.usdc",
    "ReferenceModel.usda",
    "BaseModel.usda",
    "card.png"
])
```
After some debugging , I noticed the list entries in the python 3 version have the path to temp folder which is really bizarre:
```
['Users/sabrih/AppData/Local/Temp/tmpp7c6l4yj/MyAwesomePackage.usdc', 
'Users/sabrih/AppData/Local/Temp/tmpp7c6l4yj/ReferenceModel.usda', 
'Users/sabrih/AppData/Local/Temp/tmpp7c6l4yj/BaseModel.usda', 
'Users/sabrih/AppData/Local/Temp/tmpp7c6l4yj/card.png']
```

VS the python 2 version doesn't have this temporary paths:

`['MyAwesomePackage.usdc', 'ReferenceModel.usda', 'BaseModel.usda', 'card.png']`

To me it looks like something to do with `UsdZipFile`.

Reply:
Hmm, if that's the case, then it seems like that might be an issue with core USD?

Can you reproduce this in a non-Maya context, and if so, could you open an issue against the core USD GitHub project, if so?

Reply:
@mattyjams  see 28f6222c0e0532115fc03c1da4fa2812313aa79b

I ran part of the test in question in a non-Maya context both in Python 2 and 3.  In my test I am **not** calling `cmds.usdExport `and as it shown below in both cases fileNames do contain temporary paths in the names. So I don't think this has anything to do with UsdZipFile. 

I suspect it either has something to do with `os.path.abspath` or the call to `usdExport`. For now I have disabled this assert for Python 3 until further investigation.

Python 3
```
Python 3.7.7 (tags/v3.7.7:d7c567b08f, Mar 10 2020, 10:41:24) [MSC v.1900 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import os
>>> from pxr import Usd, UsdGeom
>>> usdFile = os.path.abspath('C:\\Users\\sabrih\\AppData\\Local\\Temp\\tmpjwhzno7b\\MyAwesomePackage.usdz')
>>> zipFile = Usd.ZipFile.Open(usdFile)
>>> fileNames = zipFile.GetFileNames()
>>> print(fileNames)
['Users/sabrih/AppData/Local/Temp/tmpjwhzno7b/MyAwesomePackage.usdc', 'Users/sabrih/AppData/Local/Temp/tmpjwhzno7b/ReferenceModel.usda', 'Users/sabrih/AppData/Local/Temp/tmpjwhzno7b/BaseModel.usda', 'Users/sabrih/AppData/Local/Temp/tmpjwhzno7b/card.png']
```

Python 2
```
Python 2.7.15 (v2.7.15:ca079a3ea3, Apr 30 2018, 16:30:26) [MSC v.1500 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import os
>>> from pxr import Usd, UsdGeom
>>> usdFile = os.path.abspath('C:\\Users\\sabrih\\AppData\\Local\\Temp\\tmpjwhzno7b\\MyAwesomePackage.usdz')
>>> zipFile = Usd.ZipFile.Open(usdFile)
>>> fileNames = zipFile.GetFileNames()
>>> print(fileNames)
['Users/sabrih/AppData/Local/Temp/tmpjwhzno7b/MyAwesomePackage.usdc', 'Users/sabrih/AppData/Local/Temp/tmpjwhzno7b/ReferenceModel.usda', 'Users/sabrih/AppData/Local/Temp/tmpjwhzno7b/BaseModel.usda', 'Users/sabrih/AppData/Local/Temp/tmpjwhzno7b/card.png']
```